gflownet.py

```diff
@@ -117,7 +123,9 @@ def make_lr_scheduler(optimizer, config):
             return lr_scheduler
 
         if new_model:
-            self.model = self.model_class(self.config)
+            self.model = self.model_class(
```

Comment:
with the choice of moving manip2policy, we'll see what will be the minimum arguments that model_class ie policy network must have